docs/Audio.md

```diff
@@ -29,12 +29,62 @@ Determines if a referenced track is currently playing. Requires a valid referenc
 
 Ends the currently playing BGM. Will restart if resumed.
 
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.)
+
 ### Suspend BGM
 
 Suspends the currently playing BGM. If resumed, will continue from the time it was suspended.
 
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.)
+
 ### Resume BGM
 
+Will resume the music where it left off, with a fade in. The fade duration is set in the BGM controller's parameters.
```

Comment:
Maybe specify `where suspended` or something to that effect to make it clear the two nodes are related.

Reply:
Will do, thanks for the suggestion!